### PR TITLE
Add new logging docs

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
## PR Summary

Add new section on getting the payloads of the messages. Keep in mind, the "old way" is still needed as it will still give us:

vscode-powershell logs
Start up logs (from Start-EditorServices.ps1)
PSES logs (without the payloads)

## PR Checklist

Note: Tick the boxes below that apply to this pull request by putting an `x` between the square brackets.
Please mark anything not applicable to this PR `NA`.

- [x] PR has a meaningful title
- [x] Summarized changes
- [ ] PR has tests (N/A)
- [x] This PR is ready to merge and is not work in progress
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -185,6 +180,30 @@ Logs provide context for what was happening when the issue occurred.
   vscode-powershell@microsoft.com. Please still open an issue though
   so we can track the work &mdash; other users may have the same issue.
 
+#### Provide Language Server Protocol payload logs
+
+The PowerShell extension works mostly from sending and receiving messages from [PowerShell Editor Services](httos://github.com/PowerShell/PowerShellEditorServices).
+If we need more information, we may ask to see the payloads of these messages. To do this:
```
Comment (rjmholt) [inline docs/troubleshooting.md]:
```suggestion
In some cases, getting to the bottom of a bug will require looking at the payloads of these messages. To do this:
```

Comment (rjmholt) [reply docs/troubleshooting.md]: Makes it sound less like we're adversely exercising a contact

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -185,6 +180,30 @@ Logs provide context for what was happening when the issue occurred.
   vscode-powershell@microsoft.com. Please still open an issue though
   so we can track the work &mdash; other users may have the same issue.
 
+#### Provide Language Server Protocol payload logs
+
```
Comment (SydneyhSmith) [inline docs/troubleshooting.md]: Should we specify that this only applies to certain versions of the extension i.e. if you have 2019-10 or above?

**Comment by TylerLeonhardt** [comment]
@rjmholt 

> Is there any information that is no longer relevant and needs to come out (like about logs on the filesystem)?

No we will still use that as the first attempt to diagnose the problem. That gives us:

vscode-powershell logs
Start up logs (from Start-EditorServices.ps1)
PSES logs (without the payloads)

File: docs/troubleshooting.md (hunk 2 above)
Comment (TylerLeonhardt) [reply docs/troubleshooting.md]: added!

**Comment by TylerLeonhardt** [comment]
We can wait to merge this until the release. Just to be safe.

**Pull request merged**